narwhals/dataframe.py

```diff
@@ -884,34 +922,35 @@ def head(self, n: int) -> Self:
 
     def drop(self, *columns: str | Iterable[str]) -> Self:
         """
-        Remove columns from the dataframe.
+        Remove columns from the LazyFrame.
+
+        This is a test line to check if changes are actually being pushed.
 
```

Comment:
😄 it's definitely being pushed

but you see here that it shows a lot of changes for the `drop` method? there shouldn't be any changes here